sample-apps/s3-java/5-invoke.sh

```diff
@@ -9,7 +9,7 @@ if [ ! -f event.json ]; then
 
 fi
 while true; do
-  aws lambda invoke --function-name $FUNCTION --payload file://event.json out.json
+  aws lambda invoke --function-name $FUNCTION --payload file://event.json out.json --cli-binary-format raw-in-base64-out
```

Comment:
The `--cli-binary-format` option is not available in the AWS CLI v1, and causes an error in that version, which is still supported. I think you can avoid using this option by ditching the `file://` syntax and loading the event into a variable:
```
PAYLOAD=$(cat event.json)
aws lambda invoke --function-name $FUNCTION --payload "$PAYLOAD" out.json
```
It works for me on CLI v1. Can you try with v2?

Reply:
Tested using aws-cli/2.7.29; it's not working. 

```
An error occurred (InvalidRequestContentException) when calling the Invoke operation: Could not parse request body into json: Could not parse payload into json: Invalid UTF-8 middle byte 0x28
 at [Source: (byte[])"E�(����^�*'�ǚ�ݞ���5��9��rЫy�^t��z���*.�ǚ��7z���8�{m6�M��=4�Mv㞙z���W��*'�Zު笴��jg�z�,���x��u�޲�{�ny۞u�޲�'��%zg���@�2�p[,��o�je>�B�:�
��*�'�_w��ܠ��!bq�/q��.�_��C�l
�0��]�#q�v���,�{7���n�$z�{s�u�i�hn7��G��v�w�����7��ڦx,�7�׎�y6�w]����ι�u�o{�W\}�=o�7��슉�vǪ���z�4�N��COwn��t���U�슉�����^�حʚ�ȩjRd� 4�!s-=�(r�"; line: 1, column: 4]
```